### DISCUSSION TITLE
Cheat sheet for API rules?

### DISCUSSION THREAD

**Discussion by FrenchFryInteractive** [pockethost/pockethost#277] [Q&A]
Does anyone have a cheat sheet for a newbie? Like how do i ensure that someone who is an admin can do admin stuff? and how do i use collections data in api rules?

**Comment by benallfree** [comment]
Hello, I recommend asking this question in the PocketBase discussion board. This forum is for PocketHost and related issues. Anything about how to use PocketBase itself is probably going to get better answers over in their forum.
